### PR TITLE
Improve compatibility with older browsers

### PR DESCRIPTION
**Pull request by microbit-robert**
Avoid using top-level await and target es2017 to improve compatibility with older browsers.

### CODE REVIEW

**Comment by microbit-robert** [comment]
Needs BrowserStack testing.

File: src/__tests__/i18n.test.ts [hunk 1]
```diff
@@ -32,11 +32,11 @@ describe('Initialization tests', () => {
         search: '',
       },
     }));
-    const i18n = await import('../i18n');
-    const getText = get(i18n.t);
+    const { t } = await import('../i18n');
+    await new Promise(res => setTimeout(res, 20));
```
[the remark this answers is not in the document]
Comment (microbit-grace) [reply src/__tests__/i18n.test.ts:36]: Maybe this? 

`import { waitLocale } from 'svelte-i18n';`
```suggestion
    await waitLocale();
```

Found `waitLocale` underneath `$isLoading` in [Getting Started svelte-i18n doc](https://github.com/kaisermann/svelte-i18n/blob/main/docs/Getting%20Started.md#4-initializing)

[the remark this answers is not in the document]
Comment (microbit-grace) [reply src/__tests__/i18n.test.ts:36]: Nevermind, doesn't work 😞

[the remark this answers is not in the document]
Comment (microbit-robert) [reply src/__tests__/i18n.test.ts]: @microbit-grace Good call. It works if you export `waitLocale` from i18n.ts [bc675ff](https://github.com/microbit-foundation/ml-trainer/pull/227/commits/bc675ffe768bb537a5822d18827b6481412c91a8).

[the remark this answers is not in the document]
Comment (microbit-grace) [reply src/__tests__/i18n.test.ts]: Ohh, yay!

**Review by microbit-grace — COMMENTED** [review]
Code-wise looks good to me. I don't have access to BrowserStack and Matt might want to comment so will leave it for him to review

**Comment by microbit-robert** [comment]
Performance metrics will take a hit since we show nothing until the languages are loaded.

**Review by microbit-matt-hillsdon — APPROVED** [review]
Looks good, let's merge. I'll do a bit more testing of the recent changes before a release.

**Pull request merged**